Resources/views/Tree/tree.html.twig

```diff
@@ -10,7 +10,7 @@ file that was distributed with this source code.
 #}
 
 {% set routing_default_values = {} %}
-{% for default in routing_defaults %}
+{% for default in tree_configuration['routing_defaults'] %}
```

Comment:
afaik this could also be written as tree_configuration.routing_defaults (same for the fields below)